src/Commands/Auth/WhoamiCommand.php

```diff
@@ -17,8 +25,23 @@ class WhoamiCommand extends TerminusCommand
      *   Responds with the email of the logged-in user
      * @usage terminus auth:whoami -vvv
      *   Responds with the current session and user's data
+     * @field-labels
+     *   firstname: First Name
+     *   lastname: Last Name
+     *   email: Email
+     *   id: ID
+     *
+     * @return string
      */
-    public function whoami() {
+    public function whoAmI() {
+        $auth = new Auth();
+        if ($auth->loggedIn()) {
+            $user = Session::getUser()->fetch()->serialize();
+            return ResultData::message(new RowsOfFields([$user,]))->getMessage();
```

Comment:
@TeslaDethray You can use the output formatter object directly, but I suspect this could be much better with an enhancement.  What would your ideal API look like?  Maybe a log message?
```
$this->log()->debug('The data you want to see is: {table}', ['table' => RowsOfFields(...)]);
```
I could make something like that happen. LMK how you'd like it to work, and I'll put it together.